app-config.yaml

```diff
@@ -57,6 +57,18 @@ proxy:
     changeOrigin: true
     secure: true
 
+  '/sonarqube':
+    target: https://sonarcloud.io/api
```

Comment:
Should use `${SONARQUBE_URL}` instead of the hardcoded url.

Reply:
the e2e fails without a real value, so we left the default one